source/layouts/api_layout.html.erb

```diff
@@ -10,7 +10,10 @@
         <% end %>
       </ul>
     </li>
-    <li><%= sidebar_link 'Search API', '/apis/search-api.html' %></li>
+    <li><%= sidebar_link 'Search API', '/apis/search/search-api.html' %></li>
+    <ul>
```

Comment:
I feel like this ul should be nested inside the li tag if it's a submenu item. :)